var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -31,10 +31,10 @@ class Flex(AutotoolsPackage):
     homepage = "https://github.com/westes/flex"
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
-    version('2.6.3', 'a5f65570cd9107ec8a8ec88f17b31bb1')
```

Comment:
I had the same thoughts. The problem is/was scotch. I tried building scotch with flex 2.6.3, but it didn't look any better.